src/main/java/com/nutomic/syncthingandroid/fragments/SettingsFragment.java

```diff
@@ -297,6 +318,22 @@ public boolean onPreferenceChange(Preference preference, Object o) {
         return true;
     }
 
+    @Nullable
+    private String formatWifiNameList(Set<String> ssids) {
+        if (ssids != null) {
```

Comment:
It actually was in my case, because I had all WiFi enabled in the old version where `null` was used. Can remove that.